R/epidatacall.R

```diff
@@ -86,9 +87,9 @@ request_arguments <-
     for (name in names(all_params)) {
       v <- all_params[[name]]
       if (!is.null(v)) {
-        if (inherits(v, "EpiRange")) {
+        if (test_class(v, "EpiRange")) {
           formatted_params[[name]] <- format_item(v)
-        } else if (is.list(v)) {
+        } else if (test_list(v) || test_vector(v)) {
```

Comment:
1. This looks a little weird because lists are already vectors.
2. Also, "scalars" are vectors, so I don't see us getting to the else branch below.

I think point 2. might save us from `format_item` not working correctly on a length>1 Date vector, but from the last branch in `format_item` is looks like it was intended to work on vectors.  (But we would probably break on the weird usage `list(my_epirange, my_additional_date_vector)`

`format_list` and `format_item` are a little unclear in this vectorized world.

I'll try to update in light of above.